### ISSUE TITLE
Spacing, indentation, and whitespace

### ISSUE THREAD

**Issue by dioni21** [rodjek/puppet-lint#651]
According to this [style guide](https://docs.puppet.com/puppet/4.9/style_guide.html#spacing-indentation-and-whitespace):

The following file should give 6 warnings.  But none is given.

```
class test {
  file { '/tmp/good': ensure => file; }
  # space between title and colon
  file { '/tmp/bad1' : ensure => file; }
  # no spaces
  file{'/tmp/bad2': ensure => file; }
  file {'/tmp/bad3': ensure => file; }
  file{ '/tmp/bad4': ensure => file; }
  # too many spaces
  file   { '/tmp/bad5': ensure => file; }
  file {   '/tmp/bad6': ensure => file; }
}
```

> puppet-lint --no-autoloader_layout-check test.pp

Sorry, no patch...

**Comment by rothsa** [comment]
working on tests here: https://github.com/rothsa/puppet-lint/tree/spacing_warnings, but there's a lot of them. Will make a pull when the fixes are done.
